### PR TITLE
Removed info message display for readonly users

### PR DESCRIPTION
**Pull request by vedangimittal**
- Removed display of information message when readonly user is logged in
- Defect: https://jazz07.rchland.ibm.com:13443/jazz/web/projects/CSSD#action=com.ibm.team.workitem.viewWorkItem&id=635517

### CODE REVIEW

**Comment by Nikhil-Ashoka** [comment]
Keep the heading short

**Review by SteffiAntony — APPROVED** [review]

**Review by Nikhil-Ashoka — CHANGES_REQUESTED** [review]
Not convinced with this change. It will only cover the read-only user, If the operation fails for some reason, we would still get this info message. Need a better way to solve the issue.

**Comment by Nikhil-Ashoka** [comment]
Screenshot 2024-08-22 at 11 20 56 AM
The API for shut down is failing, but the info toast to reload the browser is still displayed. We should only show this toast message when the API is giving success

File: src/store/modules/Operations/ControlStore.js [hunk 1]
```diff
@@ -84,45 +88,55 @@ const ControlStore = {
     },
     async serverPowerOn({ dispatch, commit }) {
       const data = { ResetType: 'On' };
-      dispatch('serverPowerChange', data);
+      const displayInfo = await dispatch('serverPowerChange', data);
       await checkForServerStatus.bind(this, 'on')();
       commit('setOperationInProgress', false);
       dispatch('getLastPowerOperationTime');
```
Comment (nabilpandiyalayil) [inline src/store/modules/Operations/ControlStore.js]: We won't be getting the toast immediately when login as service user. checkForServerStatus takes 5 mins to resolve. Better to offload line 92 to 94 to a new async function.

**Review by nabilpandiyalayil — CHANGES_REQUESTED** [review]

File: src/store/modules/Operations/ControlStore.js [hunk 2]
```diff
@@ -82,47 +86,57 @@ const ControlStore = {
           throw new Error(i18n.t('pageRebootBmc.toast.errorRebootStart'));
         });
     },
-    async serverPowerOn({ dispatch, commit }) {
-      const data = { ResetType: 'On' };
-      dispatch('serverPowerChange', data);
-      await checkForServerStatus.bind(this, 'on')();
+    async powerOps({ dispatch, commit }, { arr, value }) {
+      await checkForServerStatus.bind(arr)(value);
```
Comment (nabilpandiyalayil) [inline src/store/modules/Operations/ControlStore.js]: this should be `await checkForServerStatus.bind(arr, value)()`

**Review by nabilpandiyalayil — CHANGES_REQUESTED** [review]

**Review by nabilpandiyalayil — APPROVED** [review]

**Review by Nikhil-Ashoka — APPROVED** [review]

**Pull request merged**